portal-impl/src/com/liferay/portal/layoutconfiguration/util/RuntimePageImpl.java

```diff
@@ -142,9 +109,23 @@ protected StringBundler doDispatch(
 			String templateId, String content, String langType)
 		throws Exception {
 
+		LayoutTemplateLocator layoutTemplateLocator = new LayoutTemplateLocator(
+			templateId);
+
+		if (langType == null) {
+			langType = LayoutTemplateLocalServiceUtil.getLangType(
```

Comment:
It was there in front of our eyes, nice!

Reply:
Yeah, that method saved our lives :sweat_smile: